packages/typespec-client-generator-core/src/interfaces.ts

```diff
@@ -479,11 +479,19 @@ export interface SdkQueryParameter extends SdkModelPropertyTypeBase {
   collectionFormat?: CollectionFormat;
   serializedName: string;
   correspondingMethodParams: SdkModelPropertyType[];
+  explode?: boolean;
 }
 
 export interface SdkPathParameter extends SdkModelPropertyTypeBase {
   kind: "path";
+  /**
+   * @deprecated This property is deprecated. Use `allowReserved` instead.
+   * @param serviceParam
+   */
   urlEncode: boolean;
+  explode?: boolean;
```

Comment:
Why explode/allowReserved are defined as nullable?  What if this value is null?

Seems we got this value from the compiler, @timotheeguerin, what is the meaning of null for those properties?

Reply:
I thought the compiler was always setting it. But default explode is false default style is simple. Default is not allow reserved: true